### ISSUE TITLE
Should the model.dragIndicatorValueProperty be settable?

### ISSUE THREAD

**Issue by samreid** [phetsims/center-and-variability#196]
From https://github.com/phetsims/center-and-variability/issues/129, we saw that `model.dragIndicatorValueProperty` can be set to a location with no soccer balls, which makes it look like this:

<img width="243" alt="image" src="https://github.com/phetsims/center-and-variability/assets/679486/b477033d-da42-497c-ab74-dd0d41a7f3f8">

I suspect that is undesirable. But it's unclear what the desired API should be.  Should clients be able to get or set which stack the arrow is over?  If not, how do we keep it stateful but non-interoperable?

**Comment by matthew-blackman** [comment]
@samreid @marlitas and I discussed this, and we think it's okay for the client to place the drag indicator arrow where they want. If they don't like where it is, they can place it back easily.

We still need to link the position of the drag indicator node to the correct property, and then we feel this issue can be closed.

**Comment by samreid** [comment]
I observed that the drag indicator arrow is already `link`ed to the dragIndicatorValueProperty and that changing `centerAndVariability.medianScreen.model.DragIndicatorModel.dragIndicatorValueProperty` in studio updates the location of the arrow. I observed a tandem problem which I corrected with https://github.com/phetsims/center-and-variability/commit/4171fb6b63b194022ea8215e4d1998b3a10c072a.  Closing.
